### PR TITLE
chore: minor code refactor

### PR DESCRIPTION
**Pull request by jamesgeorge007**
Minor code refactor to eliminate code redundancy.

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ try {
       runCommand("git", ["branch"])
         .split("* ")[1]
         .split(" ")[0] + (IS_DEV_MODE ? " - DEV MODE" : "");
-    if (version.variant === "" || version.variant === "master")
+    if (["", "master].includes(version.variant))
```
Comment (liyasthomas) [inline build.js]: You've a typo here

Comment (jamesgeorge007) [reply build.js]: oops :sweat_smile:

**Comment by liyasthomas** [comment]
Is there any other upcoming pulls?

**Comment by jamesgeorge007** [comment]
For sure :+1:

**Comment by liyasthomas** [comment]
Then its better to merge along those PRs. *(minor changes are being excluded in PRs)*

**Comment by jamesgeorge007** [comment]
There're couple of instances were `indexOf !== -1` is used to check for existence instead of `includes()`.
I shall make those changes herewith, after that I gues we're good to land this one :thinking:

**Comment by jamesgeorge007** [comment]
@liyasthomas hope we're good to land this one :thinking:

**Comment by liyasthomas** [comment]
Ping me when you're done refactoring

**Comment by jamesgeorge007** [comment]
All these caught my eye at a quick glance. It seems that I'm done with it :tada:

**Pull request merged**